mkdocs.yml

```diff
@@ -124,8 +124,8 @@ markdown_extensions:
       alternate_style: true
   - fancyboxmd
   - pymdownx.emoji:
-      emoji_index: !!python/name:materialx.emoji.twemoji
-      emoji_generator: !!python/name:materialx.emoji.to_svg
+      emoji_index: !!python/name:material.extensions.emoji.twemoji
+      emoji_generator: !!python/name:material.extensions.emoji.to_svg
```

Comment:
The `materialx` is deprecated and this fixes a warning.

https://squidfunk.github.io/mkdocs-material/setup/extensions/#recommended-configuration